src/Abstractions/NexusMods.Abstractions.GameLocators/Stores/Steam/SteamLocatorResultMetadata.cs

```diff
@@ -26,4 +26,8 @@ public record SteamLocatorResultMetadata : IGameLocatorResultMetadata
     
     /// <inheritdoc />
     public IEnumerable<LocatorId> ToLocatorIds() => ManifestIds.Select(m => LocatorId.From(m.ToString()));
+
+    public AbsolutePath? ProtonPrefixDirectory { get; init; }
```

Comment:
This should be documented as the path to the 'WINEPREFIX' used by the Proton distribution from Steam. If it was me, I'd call it `WinePrefixDirectory` as it's technically the most correct; and just that it's used by proton in the docs, but it works either way.

Mostly keeping that in mind for the non-Windows users; not everyone reading this code may know exactly what's going on.